src/ussd_chw.js

```diff
@@ -286,7 +286,7 @@ go.app = function() {
                 self.im.user.set_answer("operator", identity);
                 self.im.user.set_answer("operator_msisdn", operator_msisdn);
 
-                return sbm.is_identity_subscribed(identity.id, [/prebirth/]);
+                return sbm.is_identity_subscribed(identity.id, [/prebirth\.hw_full/]);
```

Comment:
You can leave this one as it was. The chw line (for 'community health worker') just subscribes them to a set with the same content as the public line.